src/libopensc/ctx.c

```diff
@@ -120,6 +120,7 @@ static const struct _sc_driver_entry internal_card_drivers[] = {
 	{ "openpgp",	(void *(*)(void)) sc_get_openpgp_driver },
 	/* The default driver should be last, as it handles all the
 	 * unrecognized cards. */
+	{ "jpki",	(void *(*)(void)) sc_get_jpki_driver },
```

Comment:
please move this change one line up as the comment is meant for the default driver (next line).